detox/ios/Detox/Actions/NSObject+DetoxActions.h

```diff
@@ -18,6 +18,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)dtx_longPressAtAccessibilityActivationPoint;
 - (void)dtx_longPressAtAccessibilityActivationPointForDuration:(NSTimeInterval)duration;
 - (void)dtx_longPressAtPoint:(CGPoint)point duration:(NSTimeInterval)duration NS_SWIFT_NAME(dtx_longPress(at:duration:));
+- (void)dtx_longPressAtPoint:(CGPoint)normalizedPoint duration:(NSTimeInterval)duration thenDragToElement:(NSObject*)target normalizedTargetPoint:(CGPoint)normalizedTargetPoint withSpeed:(CGFloat)speed thenHoldForDuration:(NSTimeInterval)lastHoldDuration
```

Comment:
Is this the top-level native-side API? Can it be changed to something that conveys there will be a drag&drop done? I think the initial long-press should be a side-effect of a function with a more descriptive name, not the other way around... wdyt? @alon-ha

Reply:
The naming convention is similar to have Apple names their method in XCUITest.

Reply:
The native methods will be named in a similar fashion to XCUITest so that we can have interchangeable driver eventually.

Reply:
As @LeoNatan said we decided to go with a convention similar to Apple API
The call from the JS side will be something like this:
`await element(by.id('elementToDrag')).longPressAndDrag(.....)`